### PR TITLE
Automate integration tests

### PR DESCRIPTION
**Pull request by EdDev**
Run the integration tests using the ovirt-stdci

### CODE REVIEW

**Comment by ifireball** [comment]
ci test please

**Comment by ifireball** [comment]
looks like I'm not whitelisted to tell CI what to do in this project...

**Comment by EdDev** [comment]
ci test please

**Comment by EdDev** [comment]
@ifireball , I did issued a `git clean` in order to clean `pyc` files when running the same locally.
I guess I could replace it with pyc specific cleaning, but the git version is much nicer.

I will try without the `git clean`.

Regarding the whitelist, just tell me how to add you.

**Comment by EdDev** [comment]
It worked! :)

File: .stdci.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+Architectures:
+  - x86_64:
+      Distributions: ["el7"]
```
Comment (tyll) [inline .stdci.yml]: The commit message mentions Fedora 28, too. So please either add it here or change the commit message.

Comment (EdDev) [reply .stdci.yml]: Done

**Pull request merged**